lib/model/frames/entity.js

```diff
@@ -44,6 +44,7 @@ Entity.Def = Frame.define(
   'id',                  'entityId',
   'createdAt',           'current',
   'submissionDefId',
+  'label',
```

Comment:
Missing `creatorId` and `userAgent`, but that can be added in another PR

Reply:
will add in another PR or let whoever comes to this part of the code next add it.